examples/webgl_geometry_spline_editor.html

```diff
@@ -87,7 +87,7 @@
 
 				const planeGeometry = new THREE.PlaneGeometry( 2000, 2000 );
 				planeGeometry.rotateX( - Math.PI / 2 );
-				const planeMaterial = new THREE.ShadowMaterial( { opacity: 0.2 } );
+				const planeMaterial = new THREE.ShadowMaterial( { color: 0x000000, opacity: 0.2 } );
```

Comment:
`0x000000` is already the default. I would remove this since it's a less verbose coding style.

Reply:
For `ShadowMaterial`, `color` is used to control the color -- and intensity -- of the shadow. This helps users see that it is an option, since it is not at all obvious.

Reply:
Sounds good to me yes.